docs/guides/executor.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Executor primitive (beta)
+description: Executor is a lower-level primitive that provides more flexibility when customizing your error mitigation workflow.
+
+---
+
+# Executor primitive (beta)
+
+<Admonition type="note" title="Beta release">
+The Executor primitive is currently in beta and might not be stable. You are invited to test it and provide feedback by opening an issue in the [Samplomatic](https://github.com/Qiskit/samplomatic/issues) or [Qiskit Runtime](https://github.com/Qiskit/qiskit-ibm-runtime/issues) GitHub repositories.
+</Admonition>
+
+The Executor primitive and its supporting tools are designed for quantum information scientists to run utility-scale experiments that require more control and flexibility. Executor provides the ingredients to capture design intents on the client side, and shifts the costly generation of circuit variants to the server side, so that you can fine-tune error mitigation and other techniques without sacrificing performance. Executor's clear, composable model for execution workflows makes it easier to experiment with new techniques, reproduce results, and share methods.
+
+In this initial release, Executor support focuses mostly on Pauli twirling, noise model learning and injection, and basis changes. Other support capabilities will be added gradually over time.
+
+## Samplomatic
+
+The Executor primitive works in the context of the [Samplomatic](https://github.com/Qiskit/samplomatic/) package, which is a library that supports customized sampling randomizations.
+
+Samplomatic uses box instructions to reason about collections of circuit operations that should be treated as having a stable noise context, and uses annotations on boxes to allow users to declare and configure intent. Every box is a scope that owns a set of qubits, operations that act on those qubits, and a list of annotations. 
```

Comment:
> Samplomatic uses box instructions

We should link to the [box API](https://quantum.cloud.ibm.com/docs/en/api/qiskit/qiskit.circuit.QuantumCircuit#box). And I think it's more accurate to call it a "construct" rather than an instruction.